.env

```diff
@@ -0,0 +1,17 @@
+# Server ports
+INNER_PORT = '8179'
+OUTER_PORT = '8179'
```

Comment:
Don't we use 8080 by default?

Reply:
It's used by frontend app and using default isn't very good - with many services it'd be hard to manage, whose port is default